manufacturing-ai-suite/wind-turbine-anomaly-detection/docs/user-guide/how-to-update-config.md

```diff
@@ -0,0 +1,26 @@
+# How to update config in Time Series Analytics Microservice
+
+The Time Series Analytics Microservice provides an interactive Swagger UI at `http://<host_ip>:5000/docs`.
+
+## Accessing the Swagger UI
+
+### To view the current configuration:
+
+1. Open the Swagger UI in your browser.
+2. Locate the `GET /config` endpoint.
+3. Expand the endpoint and click **Execute**.
+4. The response will display the current configuration of the Time Series Analytics Microservice.
+
+### To restart the Time Series Analytics Microservice UDF deployment
```

Comment:
Let's rename this as "To activate the new UDF deployment package" as for a while I felt we are restarting the time series analytics microservice here.